membrane_rtc_engine/test/support/file_endpoint.ex

```diff
@@ -0,0 +1,142 @@
+defmodule Membrane.RTC.Engine.Support.FileEndpoint do
```

Comment:
I think we should make this and fake endpoint public, put it under "Testing" namespace and remove duplicate from the HLS endpoint. This way will have sth similar to membrane_core i.e. some testing utils